bin/index.js

```diff
@@ -80,17 +89,26 @@ program
 
 function initCommand(options) {
   const selectedTemplate = options.template || "basic"; // Default to 'basic' if no template is specified
+  const packageName = options.name || "quick-start-express-server"; // Default to 'quick-start-express-server' if no name is specified
```

Comment:
Add propoer validation for `name`.
Refer npm docs.
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#name

Reply:
I have added package name validation using [validate-npm-package-name](https://www.npmjs.com/package/validate-npm-package-name).